dotnet/src/SemanticKernel/Planning/SKContextExtensions.cs

```diff
@@ -1,48 +1,195 @@
 ﻿// Copyright (c) Microsoft. All rights reserved.
 
+using System.Collections.Concurrent;
 using System.Collections.Generic;
 using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.SemanticKernel.Memory;
 using Microsoft.SemanticKernel.Orchestration;
 using Microsoft.SemanticKernel.Orchestration.Extensions;
 using Microsoft.SemanticKernel.SkillDefinition;
+using static Microsoft.SemanticKernel.CoreSkills.PlannerSkill;
 
 namespace Microsoft.SemanticKernel.Planning;
 
 internal static class SKContextExtensions
 {
-    internal static string GetFunctionsManual(
+    internal const string MemoryCollectionName = "Planning.SKFunctionsManual";
+
+    internal const string ContextRegistrationKey = "Planning.SKFunctionsAreRemembered";
```

Comment:
Consider changing the name of this constant

```suggestion
    internal const string PlanSKFunctionsAreRemembered = "Planning.SKFunctionsAreRemembered";
```